charts/rd-professional-api/values.yaml

```diff
@@ -0,0 +1,29 @@
+java:
+  applicationPort: 8090
+  environment:
+    POSTGRES_HOST: ${SERVICE_NAME}-postgresql  # do not modify
+    POSTGRES_SSL_MODE: disable
+    POSTGRES_USER: dbrefdata
+    POSTGRES_PASSWORD: dbrefdata
+    JSON_CONSOLE_PRETTY_PRINT:
+    ROOT_APPENDER:
+    ROOT_LOGGING_LEVEL:
+    REFORM_SERVICE_TYPE:
+    REFORM_SERVICE_NAME:
+    REFORM_TEAM:
+    REFORM_ENVIRONMENT:
+    LOGBACK_DATE_FORMAT:
+    LOGBACK_REQUIRE_THREAD:
+    LOGBACK_REQUIRE_ALERT_LEVEL: false
+    LOGBACK_REQUIRE_ERROR_CODE: false
+    S2S_URL: 'http://service-auth-provider-api:8080'
+  # Don't modify below here
+  image: ${IMAGE_NAME}
```

Comment:
values.yaml shouldn't be templated . you should move templated values to values.preview.teamplate.yaml.

Reply:
Done